Project-Euler/Problem013.js

```diff
@@ -1,22 +1,125 @@
 /**
- * Problem 13 - Large Sum
- *
- * @see {@link https://projecteuler.net/problem=13}
- *
- * Work out the first ten digits of the sum of the given one-hundred 50-digit numbers.
- */
+ * Work out the first ten digits of the sum of the following one-hundred 50-digit numbers.
+*/
 
-/**
- * calculates the sum and returns first 10 digits.
- * @param {String} nums
- * @returns {Number}
- */
+const nums = [
```

Comment:
`nums` should be a parameter to `largeSum` rather than a global variable. I'd also argue that you should perhaps be doing the "split by lines" in the JS as well (rather than doing it by hand & hardcoding it).

Reply:
Yeah, it is a more elegant way to do it. I fixed on the new commit